.github/workflows/enforce-labels.yml

```diff
@@ -0,0 +1,13 @@
+name: Enforce PR labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
+jobs:
+  enforce-label:
```

Comment:
I'm not sure how widely used this action is, but it might be better to use a tool that automates merges instead.
For example, https://docs.mergify.com/
This will enforce rules documented here.

Reply:
It doesn't look widely used and seems like a common use case. It's what I found with a quick google, though.

I like the idea of looking at Mergify. I previously tried to set up github's built-in auto-merge support, but it wasn't quite what I wanted. I'm happy to take that on as a follow-up assigned to me.